node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -793,20 +793,24 @@ object NodeRuntime {
         synchronyConstraintChecker,
         lastFinalizedHeightConstraintChecker
       )
-      casperLoop = for {
-        engine <- engineCell.read
-        _      <- engine.withCasper(_.fetchDependencies, Applicative[F].unit)
-        _ <- Running.maintainRequestedBlocks[F](conf.casper.requestedBlocksTimeout)(
-              Monad[F],
-              rpConfAsk,
-              requestedBlocks,
-              TransportLayer[F],
-              Log[F],
-              Time[F],
-              Metrics[F]
-            )
-        _ <- Time[F].sleep(conf.casper.casperLoopInterval.seconds)
-      } yield ()
+      casperLoop = {
+        implicit val cu = commUtil
+        for {
+          engine <- engineCell.read
+          _      <- engine.withCasper(_.fetchDependencies, Applicative[F].unit)
+          _      <- Running.updateForkChoiceTips[F]
```

Comment:
Why not just use `commUtil.sendForkChoiceTipRequest` here?

Also my concern is if this will wait for peers to respond. It would be useful to test this scenario when peers are down.